uploader/src/main/resources/cfn-template.yaml

```diff
@@ -0,0 +1,139 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "A game-changing pipeline for uploading video to the internet"
```

Comment:
nitpick: Can we change the description? game-changing should be related to games, speaking of which my softball team won a lot of games this weekend: https://twitter.com/TheMobSoftball/status/864003010548117506 ⚾